### PR TITLE
Bug fix(Issue #142)

### PR DESCRIPTION
**Pull request by LaszloRobert**
Issue : Returning NAN instead of 0 for dislikes #142 
Changes:  `response.likes` or  `response.dislikes` if it is 0 then it will be interpreted as `false` and it won't reach the code to display the correct format.

### CODE REVIEW

File: Extensions/chrome/return-youtube-dislike.script.js [hunk 1]
```diff
@@ -75,7 +75,7 @@
           cLog("response from youtube:");
           cLog(JSON.stringify(response));
           try {
-            if (response.likes || response.dislikes) {
+            if (response.likes >=0 || response.dislikes >=0) {
```
Comment (SukkaW) [inline Extensions/chrome/return-youtube-dislike.script.js:78]: Why? Both `Boolean(NaN || NaN)` and `Boolean(NaN >=0 || NaN >=0)` are false.

**Review by SukkaW — CHANGES_REQUESTED** [review]

File: Extensions/chrome/return-youtube-dislike.script.js (hunk 1 above)
Comment (gitdev-bash) [reply Extensions/chrome/return-youtube-dislike.script.js:78]: NaN translates to boolean false already

File: Extensions/firefox/return-youtube-dislike.script.js [hunk 2]
```diff
@@ -71,7 +71,7 @@ function setState() {
         cLog("response from youtube:");
         cLog(JSON.stringify(response));
         try {
-          if (response.likes || response.dislikes) {
+          if (response.likes >=0  || response.dislikes >=0) {
```
Comment (gitdev-bash) [inline Extensions/firefox/return-youtube-dislike.script.js:74]: NaN translates to boolean false already

**Review by gitdev-bash — CHANGES_REQUESTED** [review]
These changes won't change anything in the function of the code.
NaN already translates to boolean false.

**Comment by Anarios** [comment]
Fixed a bit differetly, but thanks for idea.

**Pull request closed without merging**